### DISCUSSION TITLE
Cell inspector

### DISCUSSION THREAD

**Discussion by jdhughes-dev** [pyphoenix/pyphoenix-project#56] [Ideas]
We talked about this feature for flopy 3 but never implemented it. I think it may have been called "**Cell Inspector**".

If you passed a cell to the **Inspector** you maybe could get a pandas data frame or dictionary back with the data you could also just get back a *repr* like

```
cell x,y,z

grid data
top: 10
bottom: 9
horizontal area: 12.1

hydraulic data
k11: 10
k33: 1
sy: 0.2
ss: 1e-5

stress boundaries
constant head:
stress period 0: 10
stress period 1: 11

drain:
stress period 0: 9, 1000.0
stress period 1: 8, 1010.0

```

**Comment by wpbonelli** [comment]
closing as this is now issue #199
